django_vite/templatetags/django_vite.py

```diff
@@ -88,9 +88,9 @@ def generate_vite_asset(
         **kwargs: Dict[str, str],
     ) -> str:
         """
-        Generates a <script> tag for this JS/TS asset and a <link> tag for
-        all of its CSS dependencies by reading the manifest
-        file (for production only).
+        Generates a <script> tag for this JS/TS asset, a <link> tag for
+        all of its CSS dependencies, prelodemodule for the js dependencies
+        by reading the manifest file (for production only).
```

Comment:
I'm not sure what you mean with `prelodemodule for the js dependencies by reading the manifest file`. Could you explain it a bit more, or rewrite that sentence?

Reply:
Rephrased as

> Generates a <script> tag for this JS/TS asset, a <link> tag for
> all of its CSS dependencies, and a <link rel="modulepreload">
> for all js dependencies, as listed in the manifest file
> In development Vite loads all by itself.